src/ApiPort/CommandLine/AnalyzeOptions.cs

```diff
@@ -117,6 +119,17 @@ private void UpdateRequestFlags(Options options)
                     RequestFlags |= AnalyzeRequestFlags.ShowBreakingChanges;
                 }
 
+                if (options.ShowRetargettingIssues)
+                {
+                    RequestFlags |= AnalyzeRequestFlags.ShowRetargettingIssues;
+
+                    //ShowRetargettingIssues doesn't mean much without ShowBreakingChanges turned on
+                    if (!RequestFlags.HasFlag(AnalyzeRequestFlags.ShowBreakingChanges))
```

Comment:
I agree with Mike - the extra check isn't needed and makes the logic harder to follow.